docs/make.jl

```diff
@@ -19,12 +19,24 @@ makedocs(;
     authors="Pedro Xavier and Tiago Andrade and Joaquim Garcia and David Bernal",
```

Comment:
David E. Bernal Neira (my mom would get mad at me if I don;t put the Neira)

Reply:
Fixed 😄 I've included the Maciel too (for the same reason). Also, added @pedroripper.